src/providerhub/azext_providerhub/generated/_params.py

```diff
@@ -87,18 +88,18 @@ def load_arguments(self, _):
         c.argument('provider_namespace', type=str,
                    help='The name of the resource provider hosted within ProviderHub.')
         c.argument('rollout_name', type=str, help='The rollout name.')
-        c.argument('row2_wait_duration', type=str, help='The wait duration before the rollout '
-                   'begins in rest of the world two.')
-        c.argument('skip_regions', action=AddSkipRegions,
+        c.argument('rest_of_the_world_group_two', action=AddDefaultRolloutROW2,
+                   nargs='+', help='DefaultRollout rest of the world group two.')
+        c.argument('canary', action=AddCanaryRegion,
                    nargs='*', help='The canary regions to skip.')
```

Comment:
It's a breaking change on command line interface.

Reply:
noted breaking change in history